fasjsonclient/clientbuilder/__init__.py

```diff
@@ -0,0 +1,20 @@
+from fasjsonclient.openapi_client import Configuration, ApiClient, DefaultApi
+from fasjsonclient.openapi_client.rest import RESTClientObject
+
+from . import patch
+
+
+class ClientBuilder(object):
+  """Class that builder a fasjson openapi rest client object"""
+  def __init__(self,
+      baseurl='http://fasjson.example.test/fasjson',
+      principal_name='admin@EXAMPLE.TEST'):
```

Comment:
I don't think that having a test value as default value is a good thing, if the value is required it's better to have the constructor fail when it's not provided.

Reply:
make sense, I added it as "easy way to development" but I can remove those